### PR TITLE
Force LF line endings

### PR DESCRIPTION
**Pull request by Pandapip1**
This is required for `eipw` to successfully run locally.

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -1,3 +1,7 @@
 # GitHub highlighting for Solidity files
 # See https://github.com/github/linguist/pull/3973#issuecomment-357507741
 *.sol linguist-language=Solidity
+
+# Force Linux line endings on all files
+# Necessary for running eipw locally
+* text=auto eol=lf
```
Comment (SamWilsn) [inline .gitattributes:7]: Why not just for `*.md` files? Would be weird if a `*.bat` got converted to unix line endings :P

Comment (MicahZoltu) [reply .gitattributes:7]: I believe this line is pretty standard, and I don't think `.bat` files break if they get `LF` newlines.  While we *could* try to come up with a custom list of all possible files that may need special casing, I don't think we really gain anything from that.

Comment (Pandapip1) [reply .gitattributes:7]: I would prefer if all the files use the same line ending, for consistency reasons (why add unnecessary complexity?).

Comment (SamWilsn) [reply .gitattributes:7]: Alright, go for it then!

**Comment by Pandapip1** [comment]
This will need to be manually merged. ÇC: @MicahZoltu @lightclient

**Pull request merged**